relayer/cmd/generate_beacon_data.go

```diff
@@ -171,7 +172,6 @@ func generateBeaconData(cmd *cobra.Command, _ []string) error {
 			return fmt.Errorf("get initial sync: %w", err)
 		}
 		initialSync := initialSyncScale.ToJSON()
-		err = writeJSONToFile(initialSync, fmt.Sprintf("initial-checkpoint.%s.json", activeSpec.ToString()))
```

Comment:
As mentioned before, we can not use `initialSync` generated in same command together with other updates.

Reply:
How can we improve the command so that it generates all the updates? Surely this possible to implement.

Reply:
Resolved in https://github.com/Snowfork/snowbridge/pull/840/commits/7fda5f4b26e785095ead35230eb9af043ef6b5d6